.github/workflows/CI.yaml

```diff
@@ -15,6 +15,7 @@ jobs:
         pip install -r source/requirements.txt
     - name: Install LaTeX
       run: |
+        sudo apt-get update
```

Comment:
Is it just me or is there an extra space?

Reply:
just me lol

Reply:
It looks fine to me.
![Screen Shot 2020-11-28 at 9 32 16 PM](https://user-images.githubusercontent.com/11065157/100532698-31ddca00-31c1-11eb-91e1-5ee9fbac73b6.png)